detail/detail.js

```diff
@@ -3,6 +3,17 @@ import { renderDogDetail } from '../render-utils.js';
 
 const dogDetailContainer = document.getElementById('dog-detail-container');
 
```

Comment:
this will render a single dog on the detail page depending on which id is passed in the getDog function. Then we render that dog and append it to the container